config/locales/ru/admin.yml

```diff
@@ -214,21 +214,21 @@ ru:
         action: "Текст Кнопки"
         url: "Вставьте Ссылку"
         quote: "Введите цитату" # "Quote" as in a quotation from some other text or person
-        alt: "Describe the image for the visually impaired" # TODO: Translate
+        alt: "Опишите, что на изображении (нужно для сео)" 
```

Comment:
While alt text may be used by Google for SEO, its main purpose is for Accessibility, so that screen readers can read a description of the text for the visually impaired.

Reply:
I changed it to "(required for SEO and for visually impaired people)"
Never heard of screen readers myself and don't know what they are. Most people using the admin panel won't either, so adding an additional motivation (seo) seems logical.